static/src/stylesheets/amp/header/_pillars.scss

```diff
@@ -1,29 +1,24 @@
 .pillars {
-    color: $news-support-1;
+    color: #ffffff;
     font-size: 0;
+    height: $veggie-burger-medium - $gs-baseline;
     margin: 0;
     padding: 0 $gs-gutter / 2;
-    width: 100%;
 
     @include mq(mobileLandscape) {
-        padding-left: $gs-gutter;
-        padding-right: $gs-gutter;
+        padding: 0 0 0 $gs-gutter;
```

Comment:
I think it would be better to only overwrite `padding-left` and `padding-right`, because `top` and `bottom` are already `0`.